CorsixTH/Lua/window.lua

```diff
@@ -1538,9 +1538,6 @@ top-left corner of the window.
 top-left corner of the window.
 ]]
 function Window:hitTest(x, y)
-  if x < 0 or y < 0 or (self.width and x >= self.width) or (self.height and y >= self.height) then
```

Comment:
This line must've been here for some reason no? As you haven't compensated for its deletion.

Reply:
https://github.com/CorsixTH/CorsixTH/commit/cd96f856601aaf84108f1e364517872e98557b81#diff-44af6d1126177f5b59057ef070ae3e6f47ab2aee59de3669d753fa97db28695eL1235

There wasn't a reason given at the time, but maybe.

Reply:
Hmm. So it's just an orphaned line? I wonder if that if ever actually succeeds

Reply:
It constantly executes (fun!). Get rid of it :D